docs/fsharp/tour.md

```diff
@@ -21,7 +21,7 @@ There are two primary concepts in F#: functions and types.  This tour will empha
 ## How to Run the Code Samples
 
 >[!NOTE]
-Eventually, this documentation site will have the ability to run these code samples directly in your browser.  If your browser supports Silverlight, you can use [TryFsharp](http://www.tryfsharp.org/Create) for the time being.
+Eventually, this documentation site will have the ability to run these code samples directly in your browser. You can choose two way: Try [F# for Azure Notebooks](https://notebooks.azure.com/Microsoft/libraries/fsharp/html/FSharp%20for%20Azure%20Notebooks.ipynb) on Microsoft Azure Notbooks (Currently preview, free to use.) Or, if your browser supports Silverlight, you can use [TryFsharp](http://www.tryfsharp.org/Create) for the time being.
```

Comment:
Hi, I understand your suggestions. I feel TryFsharp too old by silverlight (I know currently working for new version), so I think about we can navigate how to evaluation for easy steps for better solutions.
I feel good and agree change sentence by your sample note.